### PR TITLE
Optimizes session responses

### LINKED ISSUE #11: Optimize sessions response
The sessions response needs to be optimized.

Due to the potentially very large data sets that might be yielded from the database, the representation transformation process can take a long time to finish, sometimes resulting in around a 10 second browser waiting time.

Removing the service data transformation can reduce this time by more than half and also reduce the size of the response by around half.

Furthermore, the result item limit should be increased from 100 to 500 to prevent consumers from having to perform numerous queries to obtain the entire result set.

### PR DESCRIPTION
**Pull request by mecha**
Resolves #11

### CODE REVIEW

File: src/Controller/SessionsController.php [hunk 1]
```diff
@@ -33,7 +33,14 @@ class SessionsController extends AbstractBaseCqrsController
      *
      * @since [*next-version*]
      */
-    const DEFAULT_NUM_ITEMS_PER_PAGE = 100;
+    const DEFAULT_NUM_ITEMS_PER_PAGE = 500;
```
Comment (XedinUnknown) [inline src/Controller/SessionsController.php]: This should come from configuration. In the future, we can let the admin change this using the GUI.

**Review by XedinUnknown — CHANGES_REQUESTED** [review]

File: src/Module/EddBkRestApiModule.php [hunk 2]
```diff
@@ -139,10 +139,26 @@ public function setup()
...
                 },
 
+                /**
+                 * The order to use for sessions in the REST API.
+                 *
+                 * @since [*next-version*]
+                 */
+                'eddbk_rest_api_sessions_ordering' => function (ContainerInterface $c) {
+                    return [
+                        $c->get('sql_order_factory')->make([
```
Comment (XedinUnknown) [inline src/Module/EddBkRestApiModule.php:156]: In the future, when order objects have their own factories, new constants should be used for keys.
https://github.com/Dhii/sql-interface/issues/9

File: src/Module/EddBkRestApiModule.php [hunk 3]
```diff
@@ -453,7 +469,6 @@ public function setup()
                         [
                             MapTransformer::K_SOURCE      => 'service_id',
                             MapTransformer::K_TARGET      => 'service',
-                            MapTransformer::K_TRANSFORMER => $c->get('eddbk_rest_api_service_id_transformer'),
```
Comment (XedinUnknown) [inline src/Module/EddBkRestApiModule.php:456]: Optimisation is mainly achieved by removing this transformer, which included service details with each session.

Comment (mecha) [reply src/Module/EddBkRestApiModule.php:456]: Is this a review, comment or just a statement?

File: src/Controller/SessionsController.php (hunk 1 above)
Comment (mecha) [reply src/Controller/SessionsController.php]: Correct. I will make amends.

File: src/Module/EddBkRestApiModule.php (hunk 3 above)
Comment (XedinUnknown) [reply src/Module/EddBkRestApiModule.php:456]: Just a comment to clarify.

**Review by XedinUnknown — APPROVED** [review]
Amazing. A breath of fresh air.

**Pull request merged**